mlrun/runtimes/function.py

```diff
@@ -625,6 +636,11 @@ def invoke(
 
             path = self._resolve_invocation_url(path, force_external_address)
 
+        if headers is None:
+            headers = {}
+
+        # if function is scaled to zero, let the DLX know we want to wake it up
+        headers.setdefault("x-nuclio-target", self.metadata.name)
```

Comment:
I think that's wrong
The Nuclio function name is not `self.metadata.name` but `get_fullname(function.metadata.name, project, tag)`

Reply:
done, tested and is fixed